webdriverio/webdriver/examples/deque/test/specs/swag.items.accessibility.spec.ts

```diff
@@ -0,0 +1,16 @@
+import {setTestContext} from "../helpers";
+import {LOGIN_USERS, PAGES} from "../configs/e2eConstants";
+
+describe('Swag Items Accessibility Spec', () => {
+
```

Comment:
nitpicking, but maybe remove this line break

Reply:
removed line break